Directory.Build.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+
+    <PropertyGroup>
+        <LangVersion>latest</LangVersion>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <GitHubOrganization>BiblioNexusStudio</GitHubOrganization>
```

Comment:
The settings here and below are primarily for NuGet packaging but they don't hurt to add, especially for an open-source repo.